lib/utils/Database.cpp

```diff
@@ -31,23 +24,4 @@ Database::~Database()
 {
 }
 
-bool
-Database::create(
-    const std::string& file_name,
-    const MPI_Comm comm)
-{
-    std::cout << "Creating file: " << file_name << std::endl;
```

Comment:
This does not affect the I/O behavior, but we have been printing out a statement whenever creating/opening a file. Removing this function is fine, but we probably want to keep this printing statement where it is used.